optimum/habana/transformers/models/llama/modeling_llama.py

```diff
@@ -537,23 +537,19 @@ def pre_attn_forward(
             if q_len == 1:
                 # next token
                 use_recompute = True if os.getenv("QUANT_CONFIG", "") else False
```

Comment:
this line can be removed and we can pass False in the recompute argument in this q_len==1 scenario

Reply:
Done